src/lib/chips/chips.scss

```diff
@@ -32,14 +75,43 @@ $mat-chips-chip-margin: $mat-chip-horizontal-padding / 4;
     display: block;
     margin: 0;
     margin-bottom: $mat-chip-vertical-padding;
+  }
+}
 
-    [dir='rtl'] & {
-      margin: 0;
-      margin-bottom: $mat-chip-vertical-padding;
-    }
+.mat-chip-remove {
+  position: absolute;
+  top: $mat-chip-border-width;
+  right: $mat-chip-border-width * 2;
+  width: $mat-chip-remove-size;
+  height: $mat-chip-remove-size - $mat-chip-remove-icon-offset;
+  padding-top: $mat-chip-remove-icon-offset;
+  font-size: $mat-chip-remove-font-size;
+  text-align: center;
+  cursor: default;
+}
 
-    &:last-child, [dir='rtl'] &:last-child {
-      margin-bottom: 0;
-    }
+.mat-chip-remove.mat-chip-remove-hidden {
+  display: none;
+}
+
+.mat-chip-list-container .mat-input-placeholder-wrapper {
+  top: -4px;
```

Comment:
Having super-specific negative absolute coordinates like this seems like a code smell to me. Is there a way to eliminate this, or to alternatively make it more explainable?